Frameworks/iOS/fat/MUXSDKStats.framework/Headers/MUXSDKPlayerBinding.h

```diff
@@ -22,6 +22,9 @@ typedef NS_ENUM(NSUInteger, MUXSDKPlayerState) {
     MUXSDKPlayerStateViewEnd,
 };
 
+@protocol MUXSDKPlayDispatchDelegate
```

Comment:
Added protocol and delegate to the player binding so we have a hook right before the player binding dispatches the play event